### ISSUE TITLE
Change RSU to TSU

### ISSUE THREAD

**Issue by ebocher** [orbisgis/geoclimate#555]
What about changing the name Reference Spatial Units to Topographic Spatial Units that makes more sense.

@j3r3m1 @ELSW56 @franetibe

**Comment by franetibe** [comment]
It can be just Topographic Units then.
However @j3r3m1 has used and publicized RSU in his previous communications.

**Comment by gpetit** [comment]
> It can be just Topographic Units then.

I agree, we are loosing the "generic" approach ... and all our previous papers are talking about RSU.
Why changing ? Is there a problem with RSU ?

**Comment by j3r3m1** [comment]
Topographic units sounds good. Reference spatial unit defines the generic name considering the scale of analysis independantly of the scale definition. Thus reference spatial unit may be conserved as generic term but the unit we calculate in Geoclimate should have a name defining themselves. Maybe in French would be easier ?

**Comment by gpetit** [comment]
In Geoclimate, currently the RSU is a city, a set of cities, a square, ...
I don't know (and it's a question from my side) if an arbitrary square (a cell) is "topographic".
And (set of) cities are more "administrative" than "topographic"

**Comment by j3r3m1** [comment]
Je me suis mal fait comprendre. L'idée est de conserver RSU comme nom générique (et peut donc coller avec une cellule de grille, des limites administratives, physiques, etc.). Mais le découpage du territoire que l'on propose (et les fonctions qui sont associées pour le réaliser) est "topographique". Les unités produites sont donc "Topographic Units" (TU) plutôt que "Reference Spatial Units". Par défaut les RSU utilisées sont donc les TU mais à l'avenir pourront être des cellules de grilles ou autres.

**Comment by gpetit** [comment]
Tu parles tellement mieux en français ! ;-)
+1

**Comment by ebocher** [comment]
The method to divide the space is indeed based on topographic features. I'm agree that Reference Spatial Units don't make sense because the acronym does not refer to anything concrete. Even if we decide to use grids, the partitioning information will be based on topographic elements. Moreover, in all deliverables we describe a segmentation with topographic features.

**Comment by ebocher** [comment]
>  However @j3r3m1 has used and publicized RSU in his previous communications.

RSU is definition from the previous project Mapuce and there is currently a limited set of publications on the new methods added in geoclimate. So if we must change it, it's time to do that.

**Comment by ebocher** [comment]
@j3r3m1 did you plan for fix this one ?

**Comment by j3r3m1** [comment]
OK I have done it and will push it soon. I actually renamed processes to prepare and create RSU to prepare and rename TSU. Then I create a new process createRsu which prepare and create the TSU but they are called RSU this time. I added an input param called rsuType. By default it is "TSU" but in the future we may also use "regularGrid" for example. Note that I did not set this parameter in the Geoindicator worflow yet. @ebocher could you do that (either right now but not necessary for 1.0 or in the future) ?
